### ISSUE TITLE
DroidWorks .SAN

### ISSUE THREAD

**Issue by bahstrike** [clone2727/smushplay#2]
smushplay works perfectly for Mysteries of the Sith (MOTS) cutscenes.  thank you!

Audio decoding for DroidWorks .SAN files works perfectly (at 2-channel 16-bit 22050hz).

However the video result is solid color bitmap..  _its kinda funny because the solid pink screen actually fades in Lol._
The file `DROIDS.SAN` actually results in a divide-by-zero crash.

_My smushplay usage can be found here:
https://github.com/bahstrike/smithSmushCutscene/blob/master/smithSmushCutscene.cpp_

**Comment by clone2727** [comment]
Yeah, there's some odd scaling going on with DroidWorks and (I think) Behind the Magic. I'll take a peek if/when I get a chance.
